app/code/Bulbulatory/Recommendations/Model/Table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Model;
+
+class Table extends \Magento\Framework\Model\AbstractModel implements \Magento\Framework\DataObject\IdentityInterface
+{
+    const CACHE_TAG = 'bulbulatory_recommendations_table';
+
+    protected $_cacheTag = 'bulbulatory_recommendations_table';
+    protected $_eventPrefix = 'bulbulatory_recommendations_table';
+
+    protected function _construct()
+    {
+        $this->_init('Bulbulatory\Recommendations\Model\ResourceModel\Table');

```

Comment:
Same as above, `::class` instead of string.

Reply:
@TM18

> Same as above, ::class instead of string.

Fixed!